src/lib/const.ts

```diff
@@ -119,6 +119,8 @@ export const CHAIN_CONTAINER_NAME = "chain";
 export const CHAIN_PORT = 8545;
 export const TCP_PORT_START = 7771;
 export const WEB_SOCKET_PORT_START = 9991;
+export const HTTP_PORT_START = 18080;
+export const DEFAULT_AQUAVM_POOL_SIZE = 8;
```

Comment:
That's a bit too high for one of three docker containers ran on a local machine, I think. `2` should do.

Reply:
Good. Will change